aten/src/ATen/native/native_functions.yaml

```diff
@@ -7916,8 +7916,14 @@
     CPU, CUDA: normal
 
 - func: normal.float_float(float mean, float std, int[] size, *, Generator? generator=None, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
+  dispatch:
+    CPU, CUDA: normal
+    Meta: normal_meta
```

Comment:
Ah darn. Like you pointed out in the description, meta tensors already work with these two overloads. That's because they're currently composite ops (`CompositeImplicitAutograd`), which call into `normal_()`, which has a meta function already.

We could technically explicitly write a meta kernel for it, but it wouldn't provide much benefit (you could argue that it's a bit faster, but I don't think we generally want to start writing explicit meta kernels for all of our composite ops just for the minor overhead wins).

The other problem with adding the `dispatch` entry above is that by making these two ops no longer composite, other backends no longer get them for free. `pytorch/xla` has an implementation of `normal_()` ([here](https://github.com/pytorch/xla/blob/master/xla_native_functions.yaml#L222)), which means that it gets `normal.float_float` for free. Removing the composite kernel will break the op for xla.

I probably missed that when we were talking in the previous gh issue. Sorry about that!

Reply:
no worries, i'm okay with this one pr being rejected. it's still nice that i've got a clarification on this particular bit, so it's not in vain :)